.changes/unreleased/tinymce-TINY-12013-2025-05-04.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Changed
+body: Default value for `pagebreak_separator` option to work with document converters.
```

I think this will be a little more palatable to developers who consider it a breaking change
```suggestion
body: Default value for `pagebreak_separator` option to better suit modern standards.
```